astropy/coordinates/angles/utils.py

```diff
@@ -18,7 +18,7 @@
 
 # Astropy
 import astropy.units as u
-from astropy.coordinates.representation import (
+from astropy.coordinates.representation.spherical import (
```

Comment:
I'd just leave this as is - from the perspective of `angles`, we should not care how `representations` is organized. Indeed, it would be fine to import from `astropy.coordinates` itself, though that may lead to circular import problems.

Reply:
I'll double check, but I think circular imports is why this import adds `.spherical`.

Reply:
Guess not. Removing it.